compiler+runtime/src/cpp/jank/codegen/llvm_processor.cpp

```diff
@@ -135,6 +135,18 @@ namespace jank::codegen
     {
       auto const global_ctor_fn(ctx->global_ctor_block->getParent());
       ctx->builder->CreateCall(global_ctor_fn, {});
+
```

Comment:
Please write a comment here for the future us which explains why this whole dance is needed. It's not at all obvious.